examples/java/pom.xml

```diff
@@ -220,6 +220,30 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>flink-runner_2.10</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
```

Comment:
I guess the Spark dependencies are here because they are provided in the runner and therefore they are not transitive. I'm afraid this will cause the examples pom to be a bit "busy" as more runners join...